dev/bin/knative/resources/knative-kafka.yaml

```diff
@@ -1,25 +1,54 @@
 ---
 ---
+
+# Copyright 2020 The Knative Authors
```

Comment:
Is it some legal requirement to have the license headers here? It seems to me that it is added in various places and the header is inconsistent. The year is different etc. If necessary would it be possible to have only one license for one file?

Reply:
To be clear all upstream released manifests have these headers. Here they are basically copied from the upstream. I don't think there is any issue with this, but I can be wrong about implications.

If you want to remove feel free to a custom script to remove them.